### PR TITLE
Add ADOPTERS.md

### PR DESCRIPTION
**Pull request by fabiand**
To keep track of KubeVirt adopters

Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
ADOPTERS.md was added in order to keep track of KubeVirt adopters.
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# KubeVirt Adopters
+
+Please use this file to let us know that you are using KubeVirt by adding your company name and a description of your use case to this document.
+
+Red Hat
```
Comment (phoracek) [inline ADOPTERS.md]: How do we sort our adopters?

Is the formatting "Adopter : Usecase" intended? It looks nice in plain-text. Not sure how I feel about the rendered markdown, it looks nice and weird at the same time.

Comment (slintes) [reply ADOPTERS.md]: I think a list with single lines would be nice? Like
`* Red Hat: Using it to run classic virtual machines`

Comment (fabiand) [reply ADOPTERS.md]: What about a table? Let me update.

**Comment by slintes** [comment]
I think a release note would be nice, so we don't forget to add it to notable changes for the next release, to raise awareness that this file exists?

File: ADOPTERS.md (hunk 1 above)
Comment (slintes) [reply ADOPTERS.md]: ok, or a table :wink:

**Comment by slintes** [comment]
Travis is green, no need to wait for functests for this change

/lgtm
/approve

**Pull request merged**